lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskServiceImpl.java

```diff
@@ -1214,7 +1228,8 @@ private Task claim(String taskId, boolean forceClaim)
       Instant now = Instant.now();
 
       checkPreconditionsForClaimTask(task, forceClaim);
-      claimActionsOnTask(task, userId, now);
+      claimActionsOnTask(task, userId, userLongName, now,
+              taskanaEngine.getEngine().getConfiguration().getAddAdditionalUserInfo());
```

Comment:
do we really need that boolean parameter?

Reply:
No :) Without this parameter, ownerLongName will just be set to null if additionalUserInfo is not set.